it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionAuditingSpec.scala

```diff
@@ -30,17 +31,17 @@ class SubscriptionAuditingSpec extends BaseAuditSpec with Eventually with DesStu
       deleteKnownFactsSucceeds(arn)
       createKnownFactsSucceeds(arn)
       enrolmentSucceeds(groupId, arn)
+      createAmlsSucceeds(Some(AmlsDetails(utr, "supervisory", "12345", LocalDate.now())))
+      updateAmlsSucceeds(utr, Arn(arn))
 
       val result = await(doSubscriptionRequest(subscriptionRequest(utr)))
 
       result.status shouldBe 201
 
-      eventually {
-        DataStreamStub.verifyAuditRequestSent(
-          AgentSubscriptionEvent.AgentSubscription,
-          expectedTags,
-          expectedDetails(utr))
-      }
+      DataStreamStub.verifyAuditRequestSent(
+        AgentSubscriptionEvent.AgentSubscription,
+        expectedTags,
+        expectedDetails(utr))
```

Comment:
Not quite sure what `expectedDetails` is.. Are we checking that the extra AMLS bit goes in to the audit event?